exercises/practice/yacht/lib/yacht.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Yacht do
+  @doc """
+  Calculate the score of 5 dice using the given category's scoring method.
+  """
+  @spec score(category :: String.t(), dice :: [integer]) :: integer
+  def score(category, dice) do
```

Comment:
What would you think about making the first argument an atom instead? It serves a bit as a "enum", it doesn't need to be uppercased, split, concatenated or anything, so it doesn't really need to be a string.

```suggestion
  @type category ::
          :ones
          | :twos
          | :threes
          | :fours
          | :fives
          | :sixes
          | :full_house
          | :four_of_a_kind
          | :little_straight
          | :big_straight
          | :choice
          | :yacht

  @doc """
  Calculate the score of 5 dice using the given category's scoring method.
  """
  @spec score(category :: category(), dice :: [integer]) :: integer
```

Reply:
I like that much better!